website/docs/patterns/feedback-yes-no/examples/feedback-yes-no.tsx

```diff
@@ -10,6 +10,7 @@ import Button from '@semcore/ui/button';
 import ThumbUpM from '@semcore/ui/icon/ThumbUp/m';
 import ThumbDownM from '@semcore/ui/icon/ThumbDown/m';
 import { Text } from '@semcore/ui/typography';
+import FeedbackIllustration from '@semcore/ui/illustration/feedback';
```

Comment:
Looks like macOS case-insensetive file system betrayed you

Reply:
autocomplete in IDE...  fixed